deps.ts

```diff
@@ -13,13 +14,15 @@ export {
 } from "https://deno.land/std@0.170.0/path/mod.ts";
 export {
   bold,
+  cyan,
   green,
   magenta,
   red,
   yellow,
 } from "https://deno.land/std@0.170.0/fmt/colors.ts";
 export { parse } from "https://deno.land/std@0.195.0/flags/mod.ts";
 export { TextLineStream } from "https://deno.land/std@0.170.0/streams/text_line_stream.ts";
+export * as JSONC from "https://deno.land/std@0.170.0/encoding/jsonc.ts";
```

Comment:
Why not use more recent version? https://deno.land/std@0.208.0/jsonc/mod.ts

Reply:
It's the std version we are using. I'd prefer not to mix versions of the same dependency

Reply:
In the shown block here there is already mixed version happening (0.195.0 for `parse` and 0.170.0 for `TextLineStream` - but yeah in general I agree that we'd like to stick to the same version. Let's do the version bump in a separate PR.

Reply:
https://github.com/denoland/deployctl/issues/216